### PR TITLE
Add Clear methods to lru cache

### PR DESCRIPTION
**Pull request by dakerfp**

### CODE REVIEW

**Comment by dakerfp** [comment]
I signed it!

File: lru/lru.go [hunk 1]
```diff
@@ -119,3 +119,15 @@ func (c *Cache) Len() int {
 	}
 	return c.ll.Len()
 }
+
+// Clear purges all stored items from the cache
```
Comment (bradfitz) [inline lru/lru.go]: end in a period.

**Comment by dakerfp** [comment]
Is there anything else I can do?

**Comment by bradfitz** [comment]
Squash your two commits down into one.

**Comment by dakerfp** [comment]
Done.

**Pull request merged**